admin-dev/themes/new-theme/scss/pages/product/product_page.scss

```diff
@@ -112,9 +112,9 @@ $product-page-padding-bottom: 80px !default;
   .product-header {
     padding-top: 1.25rem;
     padding-bottom: 1.25rem;
-    margin-top: -$grid-gutter-width / 2;
     margin-bottom: $grid-gutter-width / 2;
     background: $white;
+    border-bottom: 1px solid #dfdfdf;
```

Comment:
```suggestion
    border-bottom: 1px solid $color-separator;
```

![image](https://user-images.githubusercontent.com/14963751/148955861-7bcbb5df-9d1c-4634-ab0e-ec7fc0507cf3.png)

Reply:
Never use hard coded colors, it hardens theming the project (for the future dark mode for example 😄 )